electrumx/server/session.py

```diff
@@ -1437,3 +1437,24 @@ def get_payment_position(payment_queue, address):
             return [mn for mn in cache if mn['payee'] in payees]
         else:
             return cache
+
+
+class RepBannerElectrumX(ElectrumX):
+    '''TCP server that handles incoming Electrum connections for some tokens.'''
+
+    async def replaced_banner(self, banner):
+        network_info = await self.daemon_request('getinfo')
+        ni_version = network_info['version']
+        daemon_version = (ni_version[:ni_version.find("-")]).replace("v","")
+        mmrb = daemon_version.split('.')
+        daemon_subversion = 0 if len(mmrb) < 4 else mmrb[3]
+
```

Comment:
I don't like this; if your coin's code is so old it doesn't have networkinfo, please change that in daemon.py to check for your coin and call a separate function that uses getinfo to return a dict with 'version' and 'subversion' 'members.  getinfo is obsolete/removed in other bitcoind's so I don't want to make a separate function for that which would give the impression it's supported.